charts/mattermost-operator/templates/mattermost-operator/clusterrole.yaml

```diff
@@ -0,0 +1,156 @@
+{{- if and .Values.mattermostOperator.enabled .Values.mattermostOperator.rbac.create }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: {{ template "mattermost-operator.name" . }}
+  labels:
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/name: {{ template "mattermost-operator.name" . }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+rules:
```

Comment:
Did you intend to add permissions for `leases` latter, after Operator release?

Reply:
@Szymongib yeap, I was waiting for the release to update also the image and CRDs. All are up to date now :)